h2o-core/src/main/java/water/fvec/RollupStats.java

```diff
@@ -217,6 +208,24 @@ else if (c instanceof C8DChunk)
     return this;
   }
 
+  private long processUUIDchunk(Chunk c, long checksum, long start, long l) {
+    for( int i=c.nextNZ(-1); i< c._len; i=c.nextNZ(i) ) {
+      if( c.isNA(i) ) _naCnt++;
+      else {
+        try {
+          long lo = c.at16l(i), hi = c.at16h(i);
+          if (lo != 0 || hi != 0) _nzCnt++;
+          l = lo ^ 37*hi;
+        } catch (Chunk.WrongType wt) {
```

Comment:
The problem was with calling this thing from Python. Python code does not read the file, so it does not know whether the column is legit UUID column or not. The problem is detected during import; and that's when the error is reported. It propagates back to python. Do you think we need a python test as well? It would sound reasonable. I actually did try it manually, but did not bother to add something to python tests.